sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
Except its not a test only dependency, the dependency is already used in the main module. There shouldn't be any modules mentioned in `test dependencies` as `dependencies`.

I think a valid example would be if we were testing the optional `token` dependency for sms framework, where we theoretically ensure the token UI is exposed.